seed/lib/merging/merging.py

```diff
@@ -124,11 +125,22 @@ def merge_state(merged_state, state1, state2, priorities, ignore_merge_protectio
     # Calculate the difference between the two states and save into a dictionary
     can_attrs = get_state_attrs([state1, state2])
 
+    recognize_empty_columns = state2.organization.column_set.filter(
```

Comment:
Reaching for organization columns off the second PropertyState didn't feel great/clean, but I thought it was better than providing another parameter and updating the code base in multiple places.

Also note the filter criteria - table_name, recognize_empty, and is_extra_data. Given we're starting off with an Organization's columns, I thought this was specific enough, but someone else should consider and confirm.

Reply:
i don't see this as an issue since you are really looking at the organization and not the state2.